integration-tests/bats/1pk5col-ints.bats

```diff
@@ -746,8 +746,8 @@ DELIM
     dolt checkout test-branch-m
     run dolt merge test-branch -m "merge"
     [ "$status" -eq 0 ]
-    [ "${lines[6]}" = "test | 1 +" ]
-    [ "${lines[7]}" = "1 tables changed, 1 rows added(+), 0 rows modified(*), 0 rows deleted(-)" ]
+    [ "${lines[5]}" = "test | 1 +" ]
```

Comment:
What's the reason for this change? Do we need to add a `cli.Println()` somewhere for this test to pass?
Taylor recently fixed a similar issue with `status` (https://github.com/dolthub/dolt/pull/6243), wonder if this is another spot where we just need to log a newline.

Reply:
I think the output looks good, not missing any newlines so it seems like we're just formatting it slightly differently.

Reply:
Looks like we're not *quite* showing everything.

I ran the above BATS test against tip of `main` and this PR, and here is the difference in output:

`main`
```txt
   >>>>>>>
   Updating m6rh21injvf192p8lidj7ir004dao7ns..gc93v5k5eju38on6qqf3t38ktarf06e1
   commit iaqghu0g7s8gcc7b0n8p9plfk072hbjp (HEAD -> test-branch-m)
   Merge: m6rh21injvf192p8lidj7ir004dao7ns gc93v5k5eju38on6qqf3t38ktarf06e1
   Author: Bats Tests <bats@email.fake>
   Date:  Fri Jun 30 10:06:58 -0700 2023

        merge

   test | 1 +
   1 tables changed, 1 rows added(+), 0 rows modified(*), 0 rows deleted(-)
   <<<<<<<
```

this pr
```txt
   >>>>>>>
   Updating ohrrnpi8qlcuupg3gilh61pi6h00n2fp..fq2u0jc670pcgofjsmbpsqkbeao8uroe
   commit jd2e3uo0momqq1ua7ufg1tp1t6ogt57u (HEAD -> test-branch-m) 
   Author: Bats Tests <bats@email.fake>
   Date:  Fri Jun 30 10:07:15 -0700 2023
   
        merge
   
   test | 1 +
   1 tables changed, 1 rows added(+), 0 rows modified(*), 0 rows deleted(-)
   <<<<<<<
```

Looks like we're not showing the merge information. Think we need to fix that.